lib/src/client.dart

```diff
@@ -154,9 +154,12 @@ class ClientImpl implements Client, GeneratedMessageSender {
   }
 
   @override
-  @alwaysThrows
   Future<void> removeSubscription(Subscription subscription) async {
-    throw UnimplementedError;
+    if (subscription != null) {
+      String _channel = subscription.channel;
```

Comment:
Please update to `final channel`, see [effective-dart](https://dart.dev/guides/language/effective-dart/style#dont-use-a-leading-underscore-for-identifiers-that-arent-private)

Reply:
ok, got it